Formula/ThorsSerializer.rb

```diff
@@ -0,0 +1,57 @@
+class Thorsserializer < Formula
+  desc "Declarative Serialization Library (Json/Yaml) for C++"
+  homepage "https://github.com/Loki-Astari/ThorsSerializer"
+  url "https://github.com/Loki-Astari/ThorsSerializer.git",
+    :using => :git,
+    :tag => "1.5.7",
+    :revision => "c391b49a36c7432a3fadbbb3e602bba1f61d4078"
+
+  ENV["COV"] = "gcov"
```

Comment:
This is in the wrong place. I presume you want it inside `def install`.

Reply:
Yep that seems like a good idea.